hearth/lib/hearth/event_stream/middleware/handlers.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Hearth
+  module EventStream
```

Comment:
These are middleware. Why put them under event stream? Sure it's not applicable all the time (like host prefix).

Reply:
Following the pattern on middleware from things like `HTTP` where we have `HTTP::Middleware::ContentLength` (and others).